docs/website/root/manual/developer-docs/nodes/mithril-client-library-wasm.md

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_position: 5
+---
+
+import NetworksMatrix from '../../../networks-matrix.md';
+import CompiledBinaries from '../../../compiled-binaries.md'
+
+# Mithril client library WASM
+
+:::info
+
+Mithril client library WASM can be used by Javascript developers to use the Mithril network in their web applications.
+
+It is responsible for handling the different types of data certified by Mithril, and available through a Mithril aggregator:
+- [**Snapshot**](../../../glossary.md#snapshot): list and get.
+- [**Mithril stake distribution**](../../../glossary.md#stake-distribution): list and get.
+- [**Certificate**](../../../glossary.md#certificate): list, get, and chain validation.
+
+:::
+
+:::tip
+
+* For more information about the **Mithril network**, please see the [architecture](../../../mithril/mithril-network/architecture.md) overview.
+
+:::
+
+:::note Mithril networks
+
+<NetworksMatrix />
+
+:::
+
+## Resources
+
+| Node | Source repository | Rust documentation |
+|:-:|:-----------------:|:------------------:|
+**Mithril client WASM** | [:arrow_upper_right:](https://github.com/input-output-hk/mithril/tree/main/mithril-client-wasm) | [:arrow_upper_right:](https://mithril.network/rust-doc/mithril_client_wasm/index.html) |
+
+
+## Installation
+
+The Mithril client library is compatible with the following browsers:
+
+| Browser | Minimum version | Released | Tested in CI |
+| --- |:---:|:---:|:---:|
+| **Chrome** | `54` | 2016-10-12 | :heavy_check_mark: |
+| **Edge** | `79` | 2020-01-15 | - |
+| **Firefox** | `38` | 2015-05-12 | :heavy_check_mark: |
+| **Opera** | `41` | 2016-10-25 | - |
+| **Safari** | `15.4` | 2022-03-14 | - |
+| **Chrome Android** | `54` | 2016-10-19 | - |
+| **Firefox for Android** | `38` | 2015-05-12 | - |
+| **Opera Android** | `41` | 2016-10-25 | - |
+| **Safari on iOS** | `15.4` | 2022-03-14 | - |
+| **Node.Js** | `15.4.0` | 2020-12-09 | :heavy_check_mark: |
+
+
+:::warning
+
+The package is not yet released (it will be published soon on [npm](https://www.npmjs.com/)). In the meantime, you can test the librayr by compiling it from [Mithril repository](https://github.com/input-output-hk/mithril/tree/main/mithril-client-wasm).
+
+:::
+
+In your Javascript project, use `npm` to add [mithril-client]https://www.npmjs.com/package/@mithril-dev/mithril-client-wasm) library as a dependency:
+
+```bash
+npm i @mithril-dev/mithril-client-wasm
+```
+
+## Using Mithril client library
+
+Below is a basic example of how to use most of the functions exposed by the Mithril client library:
+
+```js
+import { MithrilClient } from "@mithril-dev/mithril-client-wasm"
```

Comment:
Is this example enough for now? We also have the example in the www/ directory, which includes interaction with the DOM.

Reply:
Let's keep it simple :slightly_smiling_face: